compiler/tflchef/proto/tflchef.proto

```diff
@@ -458,7 +459,7 @@ message SegmentSumOptions {
 }
 
 message UnidirectionalSequenceLSTMOptions {
-  optional Activation activation = 1 [default = NONE];
+  optional Activation activation = 1 [default = TANH];
```

Comment:
this doesn't seem to be related with `variable`

Reply:
Oh. that's my mistake. I will fix it.